packages/unlock-js/openapi.yml

```diff
@@ -1324,7 +1324,7 @@ paths:
             type: number
 
         - in: query
-          name: address
+          name: erc20Address
```

Comment:
I would not change these. Just the internal naming, that's all.

Reply:
It will not be a breaking change here we use it with the `operationId` in `unlock-app`

Reply:
It is still a breaking change externally outside of our storage service and I don't see good reason to do that so I will keep it as it is but internally refactor.

Reply:
restored